packages/flutter_tools/integration_test/commands/packages_test.dart

```diff
@@ -14,8 +14,8 @@ import 'package:flutter_tools/src/commands/packages.dart';
 import 'package:process/process.dart';
 import 'package:test/test.dart';
 
-import '../src/common.dart';
-import '../src/context.dart';
+import '../../test/src/common.dart';
+import '../../test/src/context.dart';
 
 void main() {
```

Comment:
We already retry when pub fails, this just effectively extends the timeout on those retries.
If that's what we want to do, we should just extend the timeout. Though if it doesn't work in the first 30 seconds...